ipi/pes/fileIOCalculator.py

```diff
@@ -0,0 +1,334 @@
+import os
```

Comment:
could you explain the intended purpose of this routine? I am not sure I am getting why so much code is needed, but I think I just need some explanation :-)